data/mods/No_Survivor_Armor/survivor.json

```diff
@@ -55,5 +55,6 @@
         "sleeveless_duster_survivor",
         "duster_survivor",
         "survivor_belt_notools"
```

Comment:
You need a comma on the end of this line. The only line which doesn't end with a comma in this array should be the very last one - your addition.